tests/e2e/fixtures/statsFixtures.js

```diff
@@ -462,6 +519,49 @@ const seasonTwoGames = [
     p1: 'player2',
   },
 ];
+const seasonFourGames = [
+  // Week 1
+  {
+    name: 'Hearts Week 1, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs().subtract(1, 'week').valueOf(),
+    p0: 'player1',
+    p1: 'player2',
+  },
+  // Week 2
+  {
+    name: 'Hearts Week 2, Game 1',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs().subtract(2, 'week').valueOf(),
+    p0: 'player1',
+    p1: 'player2',
+  },
+  {
+    name: 'Hearts Week 2, Game 2',
+    status: false,
+    p0Ready: true,
+    p1Ready: true,
+    passes: 0,
+    turn: 12,
+    isRanked: false,
+    result: 0,
+    updatedAt: dayjs().subtract(3, 'week').valueOf(),
```

Comment:
If the current season started two weeks ago, this is before that
```suggestion
    updatedAt: dayjs().subtract(2, 'week').add(2, 'day').valueOf(),
```